app/src/full/java/io/homeassistant/companion/android/sensors/GeocodeSensorManager.kt

```diff
@@ -58,7 +62,14 @@ class GeocodeSensorManager : SensorManager {
                     return@addOnSuccessListener
                 }
 
-                if (location.accuracy <= LocationSensorManager.MINIMUM_ACCURACY)
+                val sensorDao = AppDatabase.getInstance(context).sensorDao()
+                val sensorSettings = sensorDao.getSettings(geocodedLocation.id)
+                val minAccuracy = sensorSettings
+                    .firstOrNull { it.name == "Minimum Accuracy" }?.value?.toIntOrNull()
```

Comment:
Should we replace the string with the constant here?

Reply:
Yep!